addons/mail/models/mail_template.py

```diff
@@ -85,6 +88,26 @@ def _compute_can_write(self):
         for template in self:
             template.can_write = template in writable_templates
 
+    def _compute_is_base_template(self):
+        template_external_ids = self.get_external_id()
+        for template in self:
+            template.is_base_template = bool(template_external_ids[template.id])
+
+    @api.model
+    def _search_is_base_template(self, operator, value):
+        if operator not in ['=', '!='] or not isinstance(value, bool):
+            raise NotImplementedError(_('Operation not supported'))
+        base_templates = self.env['mail.template'].search(
+            [('active', '=', True)]
+        ).filtered(lambda t: t.is_base_template)
+
+        # remove templates that we do not want to be put in evidence
+        base_templates_blacklist = base_templates.filtered(lambda t: not t.description)
+
+        search_bool = (operator == '=') == value
+        return [('id', 'in' if search_bool else 'not in', base_templates.ids), ('id', 'not in', base_templates_blacklist.ids)]
```

Comment:
Would be better to return a domain using fields on template model and avoid additional search queries, like (did not check it was the right one :D )
```
if operator == '=' and value or operation == '!=' and not value:
    return [('active', operator, value), ('description', '!=', False)]
return [('active', operator, value), ('description', '=', False)]
```